### ISSUE TITLE
BUG : Settings & Lyrics does not open 

### ISSUE THREAD

**Issue by NiketanG** [ytmdesktop/ytmdesktop#194]
The settings and Lyrics windows don't open. Both keep showing a loading spinner forever. 
Platform : Building from source, v1.10.0 on Linux - Fedora 32

**Comment by adlerluiz** [comment]
Thank you for report, I will create a virtual machine and test

**Comment by rickinbsb** [comment]
@NiketanG can you launch the app from the terminal and see if it outputs any error?

**Comment by NiketanG** [comment]
This is the only output shown : 

```
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
```

**Comment by NiketanG** [comment]
Also, for some reason, The App doesn't have an icon (in the Dock), when launching from AppImage. Works fine when manually launched using npx

**Comment by raxetul** [comment]
I can't open settings either(from the gear button, don't know if there is an another way). Desktop environment is Gnoıme.

![resim](https://user-images.githubusercontent.com/1366577/84564113-4b04d280-ad69-11ea-8ba8-4d24fefd637e.png)

**Comment by rickinbsb** [comment]
> Also, for some reason, The App doesn't have an icon (in the Dock), when launching from AppImage. Works fine when manually launched using npx

this is because of #121

**Comment by adlerluiz** [comment]
I set up vm and discovered the problem, it will be fixed in version 1.12

**Comment by adlerluiz** [comment]
now live 1.11
